scripts/generateLocalEnv.js

```diff
@@ -40,3 +40,21 @@ fs.writeFileSync(
     .map((key) => `${key}=${environment[key]}`)
```

Comment:
Instead of copying a env file, the setup cli path is now a requirement for e2e and they generate the env files. This is the same locally and on CI